compiler/damlc/daml-stdlib-src/DA/Date.daml

```diff
@@ -38,6 +38,14 @@ deriving instance Show Month
 deriving instance Enum Month
 deriving instance Bounded Month
 
+instance Enum Date where
+  toEnum = daysSinceEpochToDate
```

Comment:
Maybe that’s just me but it seems unintuitive to me that `minBound` is not 0.

Reply:
I don't know how to change that. According to the documentation of `Enum`, `minBound` has to have the property that `pred minBound` gives a runtime error, and our minimum date is 1.1.1.

Reply:
I wasn’t trying to suggest changing `minBound`. I was suggesting to change `toEnum`.

Reply:
To just introduce an artificial `error` if you give a negative `Int`? Wouldn't the inconsistency with the range allowed in `date` be more confusing?

Reply:
Ah, we could offset the integers by `719163` so it's zero-based. That's probably what you had in mind. I'll do that.

Reply:
Right something like `toEnum x = daysSinceEpochToDate x - daysSinceEpochToDate epoch`